src/devices/frient.ts

```diff
@@ -1,24 +1,31 @@
-import fz from '../converters/fromZigbee';
-import * as exposes from '../lib/exposes';
-import {electricityMeter, onOff, ota} from '../lib/modernExtend';
+import {develcoModernExtend} from '../lib/develco';
+import {battery, electricityMeter, onOff, ota} from '../lib/modernExtend';
 import * as reporting from '../lib/reporting';
 import {DefinitionWithExtend} from '../lib/types';
 
-const e = exposes.presets;
+// NOTE! Develco and Frient is the same company, therefore we use develco specific things in here.
 
 const definitions: DefinitionWithExtend[] = [
     {
         zigbeeModel: ['EMIZB-141'],
         model: 'EMIZB-141',
         vendor: 'Frient',
-        description: 'Smart powermeter Zigbee bridge',
-        fromZigbee: [fz.metering, fz.battery],
-        toZigbee: [],
-        extend: [ota()],
-        exposes: [e.battery(), e.power(), e.energy()],
-        configure: async (device, coordinatorEndpoint) => {
+        description: 'frient Electricity Meter Interface 2 LED',
+        extend: [
+            ota(),
+            electricityMeter({cluster: 'metering', power: {divisor: 1000, multiplier: 1}, energy: {divisor: 1000, multiplier: 1}}),
+            battery(),
+            develcoModernExtend.addCustomClusterManuSpecificDevelcoGenBasic(),
+            develcoModernExtend.readGenBasicPrimaryVersions(),
+            develcoModernExtend.pulse_configuration(),
+            develcoModernExtend.current_summation(),
+        ],
+
+        configure: async (device, coordinatorEndpoint, logger) => {
             const endpoint = device.getEndpoint(2);
             await reporting.bind(endpoint, coordinatorEndpoint, ['seMetering', 'genPowerCfg']);
+            await reporting.instantaneousDemand(endpoint);
+            await reporting.readMeteringMultiplierDivisor(endpoint);
```

Comment:
I think the configure can be removed completely, both the binds and reporting are done by `electricityMeter()` and `batter()` modernExtend.

Reply:
What about endpoint specification for the modernExtend functions?

Reply:
If only endpoint 2 has the `seMetering` cluster, it should work out of the box, is this the case?

Reply:
I think that is the case. So let's see how it goes without the configuration